examples/meter/meter.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Meter Example | WAI-ARIA Authoring Practices 1.1</title>
+<!--  Core js and css shared by all examples; do not modify when using this template. -->
+<link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+<link rel="stylesheet" href="../css/core.css">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
+
+<script src="../js/examples.js" type="text/javascript"></script>
+<script src="../js/highlight.pack.js"></script>
+<script src="../js/app.js"></script>
+
+<!-- CSS and JS for this example. -->
+<link rel="stylesheet" href="css/meter.css">
+<script src="meter.js" type="text/javascript"></script>
+
+</head>
+<body>
+<nav aria-label="Related Links" class="feedback">
+  <ul>
+    <li><a href="../../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+    <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+    <li><a href="https://github.com/w3c/aria-practices/projects/10">Related Issues</a></li>
+    <li><a href="../../../#meter">Design Pattern</a></li>
+  </ul>
+</nav>
+<main>
+  <h1> Meter Example</h1>
+  The following meter example demonstrates the <a href="">meter design pattern</a>.
+
+  <section>
+    <h2 id="ex_label">Example</h2>
+
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+
+    <div id="example">
+      <p>This meter changes its value randomly every 5 seconds.</p>
+      <div role="meter" tabindex="0" aria-valuenow="90" aria-valuemin="0" aria-valuemax="128"
```

Comment:
Should the max for CPU usage be 100%?

Reply:
That makes more sense to me.

Reply:
@smhigley, change was made and it will be comitted with other changes. thanks for the catch. (I am glad this is not temperature meter because we don't know what will be the max value due to continous global warming. ;-)